lib/css/modules/_nightMode.scss

```diff
@@ -123,6 +123,12 @@
 		background-color: rgb(105, 105, 105);
 	}
 
+	#header-img-a {
```

Comment:
To make sure this only takes effect when subreddit style is disabled, you'll need to change two things:

1. Add `module.bodyClass = true;` [here](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/60be97828c22d9fb4386aa639c8039035c300762/lib/modules/styleTweaks.js#L22) - this will add the `res-styleTweaks` class to `<body>` when the styleTweaks module is enabled

1. Move this CSS to the bottom of `_styleTweaks.css` and wrap it in this condition:

```css
body.res-styleTweaks:not(.res-srstyle-enabled) {
    ...css here...
}
```